kinto/core/initialization.py

```diff
@@ -101,6 +101,15 @@ def setup_authentication(config):
     from configuration.
     """
     config.include('pyramid_multiauth')
+    settings = config.get_settings()
+
+    policies = aslist(settings['multiauth.policies'])
+    if 'basicauth' in policies:
```

Comment:
The limitation is if somebody configure another policy with the same name, but it is a edge case we can ignore I guess.